aten/src/ATen/core/PhiloxRNGEngine.h

```diff
@@ -0,0 +1,203 @@
+#pragma once
+
+// define constants like M_PI and C keywords for MSVC
```

Comment:
This code is dead right now, is that right? Because we didn't change the CPU RNG implementation in this patch. Worth a comment...